nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the Install drive {#sec-installation-booting}
 
-NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-installation is by and large the same as a BIOS installation. The
-differences are mentioned in the steps that follow.
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Assuming your computer is still running, plug in the install drive
+and restart your computer.
+
+2.   Most computers will allow to select a drive to temporary boot from by
+pressing a special key, usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd>
+or another function key. Refer to the screen or your computer's documentation.
```

Comment:
This is not a discrete step, merge down with the next step.

Reply:
i think finding the right key to press and actually pressing it can be seen as two different tasks. it's already a lot of text for these steps. merging them would be a lot of text

Reply:
Would a `::: {.note}` callout or similar construct help here?

The way I see it, the step is to boot the installation media, a *tip* or *note* can add additional details.

Reply:
very good suggestion! i'll add that